src/utils/Path.ts

```diff
@@ -42,14 +44,42 @@ export default class Path {
   constructor(input: string | Path) {
     if (input instanceof Path) {
       this.string = input.string
-    } else if (!input || input[0] != '/') {
+      return
+    }
+
+    if (!input) {
       throw new Error(`invalid absolute path: ${input}`)
-    } else {
-      this.string = sys.normalize(input)
-      // ^^ seemingly doesn’t normalize trailing slashes away
-      if (this.string != "/") while (this.string.endsWith("/")) {
-        this.string = this.string.slice(0, -1)
+    }
+
+    if (Deno.build.os == 'windows') {
+      if (!input.match(/^[a-zA-Z]:/)) {
+        if (!input.startsWith("/") && !input.startsWith("\\")) {
+          throw new Error(`invalid absolute path: ${input}`)
+        }
+        input = `C:\\${input}`
```

Comment:
shouldn't this more correctly be from PWD's drive?
```shell
> j:\
> cd this\is\my\test\dir
> pkgx bash \my\test\script\is\on\j:\test.bash
```?

Reply:
Added a comment or two

Reply:
> terminal users are in a non existent directory more often than one might think

this is surprisingly true.